app/forms/sufia/forms/permission_template_form.rb

```diff
@@ -26,7 +26,7 @@ def update(attributes)
 
         # This allows the attributes
         def grants_as_collection(attributes)
-          attributes_collection = attributes[:access_grants_attributes]
+          attributes_collection = Array.wrap(attributes[:access_grants_attributes])
```

Comment:
I don't think we want to wrap ActionController::Parameters in an array, as that will change the code path through this method.  Instead, return `[]` if the attribute isn't found.  See:

```ruby
params = ActionController::Parameters.new(foo: 'bar')
#=> <ActionController::Parameters {"foo"=>"bar"} permitted: false>
params.respond_to? :permitted?
#=> true
Array.wrap(params).respond_to? :permitted?
#=> false
```